internal/docstore/dynamodocstore/dynamo_test.go

```diff
@@ -99,12 +100,37 @@ func (verifyAs) QueryCheck(it *docstore.DocumentIterator) error {
 func TestConformance(t *testing.T) {
 	// Note: when running -record repeatedly in a short time period, change the argument
 	// in the call below to generate unique transaction tokens.
-	drivertest.MakeUniqueStringDeterministicForTesting(2)
+	drivertest.MakeUniqueStringDeterministicForTesting(4)
```

Comment:
Revert this change so that you don't need to include the unnecessary record files below as the tokens will be the same.

Reply:
Done.